composer.json

```
@@ -27,15 +27,15 @@
         }
     ],
     "require": {
-        "php": "^7.1|^8.0",
-        "illuminate/support": "^5.0|^6.0|^7.0|^8.0|^9.0",
-        "intervention/image": "^2.5",
-        "plank/laravel-mediable": "^2.4|^3.0|^4.2|^5.4"
+        "php": "^8.0|^8.1|^8.2|^8.3",
```

wouldn't `"php": "^8.0",` cover all the options here?

```suggestion
        "php": "^8.0.2",
```